index.html

```diff
@@ -208,91 +285,136 @@ <h2>
+                browsing context</a> which is) in focus by a window which is in
+                focus by the operating system's window manager queue <a>pointerlockerror</a> 
+                and <a href="https://heycam.github.io/webidl/#reject">reject</a> the promise with a
+                "{{WrongDocumentError}}" {{DOMException}}.
+              </li>
+              <li>
+                If the pointer is already locked by an element whose <a>shadow-including root</a> is a different
+                document queue <a>pointerlockerror</a> and <a href="https://heycam.github.io/webidl/#reject">reject</a> 
+                the promise with a "{{InUseAttributeError}}" {{DOMException}}.
+              </li>
+              <li>
+                If the pointer is already locked by an element whose <a>shadow-including root</a> is a different
+                document queue <a>pointerlockerror</a> and <a href="https://heycam.github.io/webidl/#reject">reject</a> 
+                the promise with a "{{InUseAttributeError}}" {{DOMException}}.
+              </li>
+              <li>
+                If any element (including this one), whose <a>shadow-including
+                root</a> is same as this element's <a>shadow-including root</a>, is
+                already locked (or pending lock) follow these steps:
+                <ol>
+                  <li>
+                    If the <a>pointerLockOptions</a> have not changed the pointer lock <a>target</a>
+                    must be updated to this element, a <a>pointerlockchange</a>
+                    event sent, and the promise must <a>resolve</a>.
+                  </li>
+                  <li>
+                    If the options have changed and the new options are not supported queue <a>pointerLockError</a>
+                    and <a href="https://heycam.github.io/webidl/#reject">reject</a> the promise with a
+                    "{{NotSupportedError}}" {{DOMException}}
+                  </li>
+                  <li>
+                    If the options have changed and the new options are supported the pointer lock <a>target</a>
+                    must be updated to this element, the new options must be used, a <a>pointerlockchange</a>
+                    event sent, and the promise must <a>resolve</a>.
+                  </li>
+                </ol>
+              </li>
+              <li>
+                If any element (including this one), whose <a>shadow-including
+                root</a> is same as this element's <a>shadow-including root</a>, is
+                already locked (or pending lock) the pointer lock <a>target</a>
+                must be updated to this element, a <a>pointerlockchange</a>
+                event sent and the promise must <a>Resolve</a>.
+              </li>
+              <li data-link-for="Document">
+                If the request was not started from an <a>engagement gesture</a> and 
```

Comment:
We should clean up the "engagement gesture" wording to use https://html.spec.whatwg.org/multipage/interaction.html#tracking-user-activation. See also [this guidance](https://docs.google.com/document/d/14wT89JZ0qeRehXGkcn3_meXxjvlHKgM9d7aJj80kQcQ/edit). But, I think it's probably better to defer that to a future PR.

Reply:
Hmm, that seems like a really unfortunate divergence... but that's a separate issue from this update, I guess.